nexus/completion_test.go

```diff
@@ -40,10 +41,11 @@ func TestSuccessfulCompletion(t *testing.T) {
 	ctx, callbackURL, teardown := setupForCompletion(t, &successfulCompletionHandler{})
 	defer teardown()
 
-	request, err := NewCompletionHTTPRequest(ctx, callbackURL, &OperationCompletionSuccessful{
-		Header: http.Header{"foo": []string{"bar"}},
-		Body:   bytes.NewReader([]byte("success")),
-	})
+	completion, err := NewOperationCompletionSuccessful("content", OperationCompletionSuccesfulOptions{})
```

Comment:
I think we ought to have a test case where the serializer is set.

Reply:
I was too lazy to add it but I'll do it before merging.